### PR TITLE
fix(pkgs): handle unhandled props and keys

### PR DESCRIPTION
**Pull request by cdaringe**
# problem statement

- some handled props are being passed thru as unhandled, and emitting warnings
- `key` is missing on an array

# solution

patch

### CODE REVIEW

File: src/components/charts/AreaChart/AreaChart.jsx [hunk 1]
```diff
@@ -65,69 +65,66 @@ class AreaChart extends React.Component {
     })
   }
 
-  renderToolTip () {
-    if (this.props.tooltipVisible) {
-      return (
```
Comment (cdaringe) [inline src/components/charts/AreaChart/AreaChart.jsx]: this diff on this from GH is totally nuts!  all that really happened here was:

- the function now receives an arg
- the negative case is tested FIRST, then returns immediately, vs wrapping the negative case in a conditional

File: src/components/charts/AreaChart/AreaChart.jsx [hunk 2]
```diff
@@ -272,8 +268,6 @@ AreaChart.defaultProps = {
   padding: 20,
   tooltipContentType: ['x', 'y'],
   tooltipVisible: true,
-  tooltipWidth: 150,
-  tooltipHeight: 40,
```
Comment (cdaringe) [inline src/components/charts/AreaChart/AreaChart.jsx]: unused props

Comment (andrewleyva) [reply src/components/charts/AreaChart/AreaChart.jsx]: 👍

File: src/components/charts/AreaChart/AreaChart.jsx (hunk 1 above)
Comment (andrewleyva) [reply src/components/charts/AreaChart/AreaChart.jsx]: 👍

**Pull request merged**